ftw/referencewidget/resources/refwidget.js

```diff
@@ -12,7 +12,7 @@
       widget.button.on("click", openOverlay);
       $(window).one("resize", resize);
 
-      widget.name = widget.button.closest(".field").data("fieldname");
+      widget.name = "";
```

Comment:
This breaks several things:
- wrong name on given values
- wrong name compared to new selected values.
- etc.